keep/parser/parser.py

```diff
@@ -242,25 +242,27 @@ def _get_action(self, action: dict, action_name: str | None = None) -> Action:
         provider = ProvidersFactory.get_provider(
             provider_id, provider_type, provider_config, **provider_context
         )
-        action = Action(
+        action = Step(
             name=name,
             provider=provider,
             config=action,
             provider_context=provider_context,
+            step_type=StepType.ACTION,
         )
         return action
 
-    def _parse_actions(self, alert) -> typing.List[Action]:
+    def _parse_actions(self, alert) -> typing.List[Step]:
         self.logger.debug("Parsing actions")
         alert_actions = alert.get("actions", [])
         alert_actions_parsed = []
         for _action in alert_actions:
             parsed_action = self._get_action(_action)
+            parsed_action.step_type = StepType.ACTION
```

Comment:
Seems redundant, as https://github.com/keephq/keep/pull/186/files#diff-d70b456d958384fcd43980d14f1c8335a30d72522715c7d9b6b7a9d02845a12dR245 already handles that, no?

Reply:
Yes. That has been handled already in the `_get_action` method. Thank you for pointing that out.